.github/workflows/acceptance-tests.yml

```diff
@@ -5,38 +5,41 @@ on:
   schedule:
     - cron: "0 4 * * *" # workflow runs every day at 04:00 AM
   workflow_call: # workflow runs after code-health
-    inputs:
-      parent-event-name:
-        required: true
-        type: string
   pull_request: # you can run a specic job in your PR using GitHub labels
     types: [ labeled ]
 
+env:
+  ALWAYS_RUN: ${{ github.event_name == 'workflow_dispatch' || github.event_name == 'schedule' || github.event.label.name == 'run-testacc' }}
+  CI: true
+  PARALLEL_GO_TEST: 20
+  ACCTEST_TIMEOUT: ${{ vars.ACCTEST_TIMEOUT }}
+  SKIP_TEST_EXTERNAL_CREDENTIALS: ${{ vars.SKIP_TEST_EXTERNAL_CREDENTIALS }}
+  TF_LOG: ${{ vars.LOG_LEVEL }}
+
 jobs:  
-  change-detection:
+  check:
     runs-on: ubuntu-latest
     permissions:
       pull-requests: read
       repository-projects: read
     outputs:
-      cluster_outage_simulation: ${{ steps.filter.outputs.cluster_outage_simulation }}
-      advanced_cluster: ${{ steps.filter.outputs.advanced_cluster }}
-      cluster: ${{ steps.filter.outputs.cluster }}
-      generic: ${{ steps.filter.outputs.generic }}
-      backup_online_archive: ${{ steps.filter.outputs.backup_online_archive }}
-      backup_snapshots: ${{ steps.filter.outputs.backup_snapshots }}
-      federation: ${{ steps.filter.outputs.federation }}
-      backup_schedule: ${{ steps.filter.outputs.backup_schedule }}
-      project: ${{ steps.filter.outputs.project }}
-      serverless: ${{ steps.filter.outputs.serverless }}
-      network: ${{ steps.filter.outputs.network }}
-      config: ${{ steps.filter.outputs.config }}
-      assume_role: ${{ steps.filter.outputs.assume_role }}
-      event_trigger: ${{ steps.filter.outputs.event_trigger }}
-      search_index: ${{ steps.filter.outputs.search_index }}
+      cluster_outage_simulation: ${{ steps.filter.outputs.cluster_outage_simulation == 'true' || github.event.label.name == 'run-testacc-cluster-outage-simulation' || env.ALWAYS_RUN == 'true' }}
```

Comment:
this is probably making it even less readable :)

Reply:
I agree it's less readable this line but I think it improves the other parts and overall it's better :-)

Reply:
for example:
 if: ${{ needs.change-detection.outputs.cluster_outage_simulation == 'true' || github.event_name == 'workflow_dispatch' || github.event_name == 'schedule' || github.event.label.name == 'run-testacc' || github.event.label.name == 'run-testacc-cluster-outage-simulation' || inputs.parent-event-name == 'release'  }}

becames:
    if: ${{ needs.check.outputs.cluster_outage_simulation == 'true' }}

Reply:
also for a value that is used in all tests but one or a few you can decide to override in that test (some declaring the env var. again) or remove entirely from the common part and define in each one.

Reply:
ok sounds good.